### PR TITLE
Website enhancements2

### PR DESCRIPTION
**Pull request by MHerbst**
Unified all internal URLs to hava similar scheme and to allow https access. I have also fixes some smaller bugs.

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -18,12 +15,19 @@
...
       {% endif %}
     </title>
 
+    <link rel="icon" href="/favicon.ico" />
+    <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" />
+
+    <link href="/feed.xml" rel="alternate" type="application/rss+xml" title="openHAB Foundation Blog" />
+    <link href="/feed.xml" rel="feed" type="application/rss+xml" title="openHAB Foundation Blog" />
+	 
+	 <link rel="canonical" href="{{ site.url }}{{ page.url | replace:'.html','' }}" />
```
Comment (kaikreuzer) [inline _includes/head.html]: could you replace the tab here?

File: blog/index.html [hunk 2]
```diff
@@ -43,7 +38,7 @@ <h1 class="page-header">Smart Home Blog
             </a>
             <hr>
             <p>{{ post.excerpt }}</p>
-            <a class="btn btn-primary" href="{{url_base}}/{{ post.url }}">Read More <i class="fa fa-angle-right"></i></a>
+            <a class="btn btn-primary" href="{{url_base}}{{ post.url }}">Read More <i class="fa fa-angle-right"></i></a>
```
Comment (kaikreuzer) [inline blog/index.html]: shouldn't the `{{url_base}}` be removed here as well?

File: _includes/head.html (hunk 1 above)
Comment (MHerbst) [inline _includes/head.html]: Removed the tab

File: blog/index.html (hunk 2 above)
Comment (MHerbst) [inline blog/index.html]: Of course. I overlooked this one.

**Comment by kaikreuzer** [comment]
Many thanks!

**Comment by kaikreuzer** [comment]
Wonderful, HTTPS (almost) works!
Two issues that I found on a first test: Please go to https://www.openhabfoundation.org/:

1. The "Blog" menu appears to go to https, but the browser switches back to http
2. The "Members" drop-down menu is not working.

Any clue what's wrong there?

**Comment by MHerbst** [comment]
I have just discovered this at the same time as you did. I first thought that it is a cache problem. But I cleaned everything and it did not help. 
Will try to find out whats wrong with it. Especially the first point is strange.

**Comment by MHerbst** [comment]
First result. The trace shows that there is a redirect when trying to access the blog:
image
Is there a redirect configured somewhere on the server?

**Comment by MHerbst** [comment]
@kaikreuzer I found the reason for the second problem. I did not change _:commonjs.html. Will create a new pull request very soon

**Comment by kaikreuzer** [comment]
> Is there a redirect configured somewhere on the server?

No. Might it be due to the trailing slash?

**Comment by kaikreuzer** [comment]
> Is there a redirect configured somewhere on the server?

This must be a general Jekyll mechanism as it generated the blog content through some plugin... Not sure how we can possibly tweat this.

**Comment by kaikreuzer** [comment]
Having said this, the memberlist is also using the same blog mechanism and this works well with https...

**Comment by MHerbst** [comment]
I just want to write right this :-). 
To me it looks like it is a configuration on Cloudflare.
I have tried it with `curl -i https://www.openhabfoundation.org/blog` and got this response:
```
HTTP/1.1 301 Moved Permanently
...
Location: http://www.openhabfoundation.org/blog/
...
Server: cloudflare-nginx
CF-RAY: 32032918799d2d35-TXL

```
This is transferred in the http header returned from the server.

**Comment by kaikreuzer** [comment]
You get the same 301 if you use `curl -i http://www.openhabfoundation.org/blog`.
But you do not get it on `curl -i https://www.openhabfoundation.org/blog/` - so this is probably where our link should point to!

**Comment by MHerbst** [comment]
Interesting ... I will change it. But still it is strange that this happens. Maybe there es some default redirect somewhere (on Cloudflare?)  that causes this problem.
The link to membershiplist works without trailing "/" ....

**Comment by kaikreuzer** [comment]
Yes, because there is a html page at http://www.openhabfoundation.org/members/membership.html.

This link also works without a trailing / and without a redirect: https://www.openhabfoundation.org/blog/index

**Comment by MHerbst** [comment]
That explains he difference :-). Hope it is solved now.

**Pull request merged**